front/app/components/admin/AddCollaboratorsModal/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+// Components
+import { Box, Button, Text } from '@citizenlab/cl2-component-library';
+import Modal from 'components/UI/Modal';
+import SeatInfo from 'components/SeatInfo';
+
+// Translation
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// hooks
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+import useSeats from 'api/seats/useSeats';
+
+// Utils
+import { isNil } from 'utils/helperUtils';
+
+interface Props {
+  showModal: boolean;
+  closeModal: () => void;
+  addModerators: () => void;
+  // Total number of seats to add, including admin seats
```

Comment:
I think, ideally, we don't want admins to be part of the list of moderators you can add. It's confusing... Admins have these rights anyway.

Reply:
In the case we decide to do that, `noOfCollaboratorSeatsToAdd` could be used.

Reply:
(Edit: Check the discussions in Slack, we want to filter admins out of the list of project moderators).

Reply:
There was prior discussion that I highlighted in slack. I'll make the change after decision is made there